### PR TITLE
update activation op on kunlun

### PR DESCRIPTION
**Pull request by tangzhiyi11**
### PR types
Function optimization

### PR changes
OPs 

### Describe
update activation op on kunlun
*test=kunlun

### CODE REVIEW

File: paddle/fluid/operators/activation_op_xpu.cc [hunk 1]
```diff
@@ -115,99 +87,198 @@ void xpu_activation_backward(const framework::ExecutionContext &ctx,
...
+        platform::errors::External("XPU constant op return"
+                                   " wrong value[%d %s] in pow op.",
+                                   r, XPUAPIErrorMsg[r]));
+    r = xpu::pow(dev_ctx.x_context(), x_data, factor_data, y_data, x->numel());
+    PADDLE_ENFORCE_EQ(r == xpu::Error_t::SUCCESS, true,
+                      platform::errors::External("XPU pow op return"
+                                                 " wrong value[%d %s].",
+                                                 r, XPUAPIErrorMsg[r]));
+
+    dev_ctx.Wait();
```
Comment (QingshuChen) [inline paddle/fluid/operators/activation_op_xpu.cc]: 判断下stream确定是否wait

Comment (tangzhiyi11) [reply paddle/fluid/operators/activation_op_xpu.cc:192]: XPUDeviceContext里貌似没有stream相关的信息，这个统一在XPUDeviceContext的Wait()方法里加上？

Comment (tangzhiyi11) [reply paddle/fluid/operators/activation_op_xpu.cc]: ok，已经fix

**Review by fuyinno4 — APPROVED** [review]

**Review by chenwhql — APPROVED** [review]
LGTM for PADDLE_ENFORCE

**Review by luotao1 — APPROVED** [review]

**Pull request merged**